build.gradle

```diff
@@ -35,6 +38,9 @@ platform {
 
 	categoryName 'TestNG Libraries'
 	categoryId 'org.testng.p2.libraries'
+	
+	useBndHashQualifiers false
+    defaultQualifier ''
```

Comment:
I'm thinking for snapshot versions like '6.9.13-SNAPSHOT', can we utilize the version qualifier that to be substituted with timestamp, e.g. to be '6.9.13.201609291640'